package.json

```diff
@@ -6,7 +6,8 @@
   "workspaces": [
     "contracts",
     "examples/contracts/*",
-    "packages-ts/*"
+    "packages-ts/*",
+    "examples/new_contracts/*"
```

Comment:
I'd create an initial commit that removes the old contracts then place the new ones under `examples/contracts`. That way the scripts can at least be reviewed as a diff and less adjustments will be needed on the CI

Reply:
sounds good! I initially was following the pattern we did for the contracts but you're right in that they should be compared against each other since we're not holding on to the old examples